mne/decoding/tests/test_time_gen.py

```diff
@@ -158,8 +175,13 @@ def test_generalization_across_time():
 
     # Test start stop training & test cv without n_fold params
     y_4classes = np.hstack((epochs.events[:7, 2], epochs.events[7:, 2] + 1))
-    gat = GeneralizationAcrossTime(cv=LeaveOneLabelOut(y_4classes),
-                                   train_times={'start': 0.090, 'stop': 0.250})
+    train_times = dict(start=0.090, stop=0.250)
+    if check_version('sklearn', '0.18'):
+        # cv = LeaveOneLabelOut()  # XXX wait for sklearn issue #6304
```

Comment:
if it's a bug in sklearn, we shouldn't try to work around it. Just report the issue there and get it fixed there.